src/basic/strv.c

```diff
@@ -789,6 +789,22 @@ int strv_extend_n(char ***l, const char *value, size_t n) {
         return -ENOMEM;
 }
 
+int strv_extend_join(char ***l, const char *lhs, const char *rhs) {
```

Comment:
hmm, one of these days this should be rework to take variadic params, and dispatch to some flavour of strjoin_real()